### DISCUSSION TITLE
Default branch

### DISCUSSION THREAD

**Discussion by Felderburg** [bcholmes/StarTrek2d20#75] [General]
So I proposed those notable NCC number additions, and noticed that the STA-Complete branch appears to be the current "working" branch, but it's not the default branch (master is, the original fork). I'm just curious: is there a reason for that?

**Comment by bcholmes** [comment]
The reason is tied to how the codebase came in to being: azjerei originally created the app, and the master branch is basically a mirror of azjerei's repo. I still have a coupl'a pull requests open with azjerei (although at this point, I should accept that they'll probably never get merged), which are branches from the original codebase. 

In theory, I can try to send more pull requests in the future, my copying stuff from sta-complete into master (or a branch of master) and submitting that back, but realistically, that probably won't happen.

Congrats on your first Pull Request!
